vm/vm.go

```diff
@@ -461,11 +506,12 @@ func (mon *monitor) waitForOutput() {
 const (
 	maxErrorLength = 256
 
-	lostConnectionCrash = "lost connection to test machine"
-	noOutputCrash       = "no output from test machine"
-	timeoutCrash        = "timed out"
-	fuzzerPreemptedStr  = "SYZ-FUZZER: PREEMPTED"
-	vmDiagnosisStart    = "\nVM DIAGNOSIS:\n"
+	lostConnectionCrash   = "lost connection to test machine"
+	noOutputCrash         = "no output from test machine"
+	executionStalledCrash = "execution stalled"
```

Comment:
This needs to be understandable by kernel developers who know nothing about syzkaller. All they will get is this string and likely not much else (no reproducer, no report).